README.md

```diff
@@ -22,12 +22,14 @@ This repository contains some of the implementations developed by DFINITY.
 
 A list of community maintained Candid libraries:
 
+* [Motoko](https://github.com/edjcase/motoko_candid)
```

Comment:
Can you append the list, so that it's in chronicle order?

Reply:
Done